src/Magnum/Audio/Test/RendererTest.cpp

```diff
@@ -34,10 +36,28 @@ struct RendererTest: TestSuite::Tester {
     explicit RendererTest();
 
     void debugError();
+    void debugDistanceModel();
+    void listenerOrientation();
+    void listenerPosition();
+    void listenerVelocity();
+    void listenerGain();
+    void speedOfSound();
+    void dopplerFactor();
+    void distanceModel();
+
+    Context _context;
 };
 
-RendererTest::RendererTest() {
-    addTests({&RendererTest::debugError});
+RendererTest::RendererTest(): _context{} {
```

Comment:
No need to call `_context{}` explicitly (it will be done implicitly by the compiler).